experimental/dds/tree2/src/core/tree/treeTextFormat.ts

```diff
@@ -25,6 +25,11 @@ import { FieldKey, NodeData } from "./types";
  * TODO: stabilize this format (probably after schema are more stable).
  *
  * This format does not include schema: typically schema would be stored alongside data in this format.
+ *
+ * @privateRemarks - A forked version of these types is available at `persistedTreeTextFormat.ts`.
```

Comment:
I didn't know about this keyword before. It's like "@remarks" that you don't want end users to see, but you want internal devs to see? Does api extractor correctly interpret this? Is it better than just putting a `//` below the doc comment?

Reply:
yeah we use it a few other places in the codebase and that made me look it up -- https://tsdoc.org/pages/tags/privateremarks/. Your description of the semantics is pretty spot on though.

Doing it this way provides the remarks in a mouseover of the type when working within the codebase, which is a bit better than `//` comments. That's the main advantage.

Reply:
Oh neat! I'll start using it then :)